schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -1472,6 +1472,12 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2019-09-01-preview/Microsoft.Peering.json#/resourceDefinitions/peerings" },
                   { "$ref": "https://schema.management.azure.com/schemas/2019-09-01-preview/Microsoft.Peering.json#/resourceDefinitions/peeringServices" },
                   { "$ref": "https://schema.management.azure.com/schemas/2019-09-01-preview/Microsoft.Peering.json#/resourceDefinitions/peeringServices_prefixes" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2020-01-01-preview/Microsoft.Peering.json#/resourceDefinitions/peerings" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2020-01-01-preview/Microsoft.Peering.json#/subscription_resourceDefinitions/peerAsns" },
```

Comment:
Subscription-level resources are not supported in this template (this template only covers resourcegroup-level resources).

If you want to include subscription-level resources, they should be added to https://github.com/Azure/azure-resource-manager-schemas/blob/master/schemas/2018-05-01/subscriptionDeploymentTemplate.json